### DISCUSSION TITLE
JWT issued more than 1200 seconds in the future.

### DISCUSSION THREAD

**Discussion by bmarker1** [OpenIDC/mod_auth_openidc#659] [Q&A]
I have the error: 'oidc_proto_validate_iat: "iat" validation failure (1629914020): JWT was issued more than 1200 seconds in the future'
Our OIDCProvider is from a different timezone which is why I believe this error is occurring.  They are GMT and we're EST.  What var do I need to set to make up for the 4 hour difference?

**Comment by zandbelt** [comment]
the provider should set a timezone agnostic timestamp as the spec dictates
